### ISSUE TITLE
Import pynput: Error

### ISSUE THREAD

**Issue by Pro100rus32** [Nuitka/Nuitka#937]
```
0.6.10.5
Python: 3.8.7 (tags/v3.8.7:6503f05, Dec 21 2020, 17:59:51) [MSC v.1928 64 bit (AMD64)]
Executable: C:\Users\andre\AppData\Local\Programs\Python\Python38\python.exe
OS: Windows
Arch: x86_64
```

**Test.py**
```
import pynput
print("Hello!")
```

Input nuitka:
`python -m nuitka --mingw64 --standalone "E:\Test\Test\Test.py"`

Output .exe:
```
Traceback (most recent call last):
  File "C:\Users\andre\TEST~1.DIS\Test.py", line 1, in <module>
  File "C:\Users\andre\TEST~1.DIS\pynput\__init__.py", line 40, in <module pynput>
  File "C:\Users\andre\TEST~1.DIS\pynput\keyboard\__init__.py", line 31, in <module pynput.keyboard>
  File "C:\Users\andre\TEST~1.DIS\pynput\_util\__init__.py", line 76, in backend
ImportError
```

**Comment by NeuralAIM** [comment]
![image](https://user-images.githubusercontent.com/75082388/105194222-6a0ee280-5b4a-11eb-91e3-6faa2bbd9cec.png)

**Comment by Pro100rus32** [comment]
![image](https://user-images.githubusercontent.com/64806740/105194295-7d21b280-5b4a-11eb-80c4-9d651376aafc.png)

**Comment by kayhayen** [comment]
This is on factory which is currently not a in a good shape, but once it stablizes it will appear in a release.

**Comment by Pro100rus32** [comment]
Thanks, it works : )

**Comment by Pro100rus32** [comment]
Oops)

**Comment by kayhayen** [comment]
Released as 0.6.11 just now.
